City-of-Pori/pori-kada
# PORIJATE-11 Additional info block

To test:

1. Sync from production
2. drush updb -y && drush fra -y && drush cc all
3. Go to edit menus and choose the waste menu FI. Add some items.
4. Go to local.pori.fi/waste/search and make sure there is the sidebar and it works responsively.
